.drone.yml

```diff
@@ -49,6 +49,25 @@ pipeline:
     secrets: [GH_TOKEN]
     commands:
       - ./scripts/deployment/deploy-documentation.sh
+    when:
+      event: [push]
+      branch: master      
+
+  deploy:
+    image: node:8.10
+    secrets:
+      [
+        eubfr_env,
+        eubfr_stage,
+        aws_access_key_id,
+        aws_secret_access_key,
+        eubfr_aws_region,
+        eubfr_content_repository,
+      ]
+    commands:
+      - npx serverless config credentials --provider aws --key $${AWS_ACCESS_KEY_ID} --secret $${AWS_SECRET_ACCESS_KEY}
```

Comment:
isn't there a risk of exposing the secrets during the drone deployment here? 🤔

Reply:
You're right, [they are](https://github.com/drone/docs/blob/master/content/usage/secrets/secrets.md), unfortunately.
I'll probably have to find another way to manage secrets in our scripts, adding WIP flag.